config/initializers/wrap_parameters.rb

```diff
@@ -5,7 +5,7 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
+  wrap_parameters format: []
```

Comment:
(Basically unrelated, but I noticed while looking at these requests in the Rails server logs that `wrap_parameters` was enabled, which I sort of hate, so I'm disabling it here.)